scripts/run_examples.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 """Run examples bundled with this repo."""
 import argparse
+import logging
```

Comment:
I'd say we don't touch this either. It's not part of the package, and it could be considered part of the testing infrastructure, which is due a few changes. There are plans to turn the examples into a pytest suite (based on some of the work done by the MNE folks), so QoL things here are not a priority.

Reply:
removed